android/AndroidManifest.xml

```diff
@@ -557,24 +559,10 @@
         <data android:scheme="https"/>
 
         <!-- #map=$zoom/$lat/$lon -->
-        <data android:host="www.openstreetmap.org" android:path="/"/>
-        <data android:host="www.openstreetmap.org" android:pathPrefix="/search"/>
-        <data android:host="openstreetmap.org" android:path="/"/>
-        <data android:host="openstreetmap.org" android:pathPrefix="/search"/>
-      </intent-filter>
-
-      <intent-filter>
-        <action android:name="android.intent.action.VIEW"/>
-
-        <category android:name="android.intent.category.DEFAULT"/>
-        <category android:name="android.intent.category.BROWSABLE"/>
-
-        <data
-          android:host="lead"
-          android:scheme="mapsme"/>
-        <data
-          android:host="lead"
-          android:scheme="mapswithme"/>
+        <data android:host="openstreetmap.org"/>
+        <data android:host="www.openstreetmap.org"/>
+        <data android:path="/"/>
+        <data android:path="/search"/>
```

Comment:
Is /search really needed? Should "/" also match /search?

Reply:
The current rule catches  `https://www.openstreetmap.org/#map=19/33.97896/50.68466` and `https://www.openstreetmap.org/search?query=test#map=19/33.97896/50.68419`. Probably `/search` is not really needed.

Reply:
The path part of a URI which must begin with a /. The path attribute specifies a complete path that is matched against the complete path in an Intent object. The pathPrefix attribute specifies a partial path that is matched against only the initial part of the path in the Intent object. The pathPattern attribute specifies a complete path that is matched against the complete path in the Intent object, but it can contain the following wildcards:

        An asterisk ('*') matches a sequence of 0 to many occurrences of the immediately preceding character.
        A period followed by an asterisk (".*") matches any sequence of 0 to many characters.

    Because '\' is used as an escape character when the string is read from XML (before it is parsed as a pattern), you will need to double-escape: For example, a literal '*' would be written as "\\*" and a literal '\' would be written as "\\\\". This is basically the same as what you would need to write if constructing the string in Java code.

    For more information on these three types of patterns, see the descriptions of [PATTERN_LITERAL](https://developer.android.com/reference/android/os/PatternMatcher#PATTERN_LITERAL), [PATTERN_PREFIX](https://developer.android.com/reference/android/os/PatternMatcher#PATTERN_PREFIX), and [PATTERN_SIMPLE_GLOB](https://developer.android.com/reference/android/os/PatternMatcher#PATTERN_SIMPLE_GLOB) in the [PatternMatcher](https://developer.android.com/reference/android/os/PatternMatcher) class.

    These attributes are meaningful only if the [scheme](https://developer.android.com/guide/topics/manifest/data-element#scheme) and [host](https://developer.android.com/guide/topics/manifest/data-element#host) attributes are also specified for the filter.
    
    https://developer.android.com/guide/topics/manifest/data-element